tasks/label_analysis.py

```diff
@@ -40,6 +44,24 @@ class LinesRelevantToChange(TypedDict):
     files: Dict[str, Optional[LinesRelevantToChangeInFile]]
 
 
+class ExistingLabelSetsEncoded(NamedTuple):
```

Comment:
What's the advantage of `NamedTuple` over using `@dataclass`?

Reply:
No idea, really. But originally this was a tuple, so I thought I'd keep it similar to a tuple.

Reply:
Apparently there are considerations to be made: https://stackoverflow.com/questions/51671699/data-classes-vs-typing-namedtuple-primary-use-cases

But the usage is quite small for this case in particular.
What I want out of this is _typing_ info, to improve code readability.
I'm OK changing to dataclass if preferred.